File: src/test/java/com/sword865/scalaJVM/javafile/ShTest.scala
package com.sword865.scalaJVM.javafile

class ShTest {

  def main(args: Array[String]): Unit = {
    val x = -1
    System.out.println(Integer.toBinaryString(x))
    System.out.println(Integer.toBinaryString(x >> 8))
    System.out.println(Integer.toBinaryString(x >>> 8))
  }

}

File: src/test/java/com/sword865/scalaJVM/javafile/ShTest.java
package com.sword865.scalaJVM.javafile;

public class ShTest {

  public static void main(String[] args) {
    int x = -1;
    System.out.println(Integer.toBinaryString(x));
    System.out.println(Integer.toBinaryString(x >> 8));
    System.out.println(Integer.toBinaryString(x >>> 8));
  }

}
